geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegionMap.java

```diff
@@ -378,6 +365,9 @@ public void close(BucketRegion bucketRegion) {
   @Override
   public Set<VersionSource> clear(RegionVersionVector rvv, BucketRegion bucketRegion) {
     Set<VersionSource> result = new HashSet<VersionSource>();
+    long nanoTime = System.nanoTime();
```

Comment:
don't directly call System.nanoTime, Instead add a startBucketClear and endBucketClear to PartitionedRegionStats and have it use the standard start code to compute time stats if they are enabled

Reply:
done